security.md

```diff
@@ -0,0 +1,12 @@
+# Security Document
+See also [code conventions](code-conventions.md); there are a few guidelines
+about security of added code there.
+
+## Reporting security issues
+Security issues are reported as anything else. Authors should indicate that
+they are security issues, so that the team knows to prioritize testing them.
```

Comment:
IMO, Serious security issues shouldn't always be reported via the issue tracker; You may want to mention to contact core team members on discord if something like a force-op is found.

Reply:
Hmm, good point. I decided not to mention it, because we don't really have anyone who is *always* available at few days notice to handle such issues. In Discord, you can't easily DM to a group. Maybe it would still make sense, have a slight delay but disclose security issues only after fixes are out.